kcl-samples

```diff
@@ -0,0 +1 @@
+Subproject commit 76d4a220bd2b8d4f608b4f39d90cdcc6edd7fad2
```

Comment:
What is this? Do we use a git submodule in this repo?

Reply:
I've been confused by that, let me get rid of the change from here and see if we use a submodule for kcl-samples in general. Thanks!